src/main/webapp/js/favorite.js

```diff
@@ -0,0 +1,45 @@
+// Fetch favorite SDKS
+async function getFavoriteSDKs(userID) {
+  const response = await fetch(`v1/users/${userID}/sdks`);
```

Comment:
Could we cache the result to avoid calling the API every time the user navigates to the home page?
Would be useful for the starSDK method too.

Reply:
This sounds like a good idea. This site seems to have good information on this but it seems like more work than I can handle in the few minutes left of my internship. https://developer.mozilla.org/en-US/docs/Web/API/Request/cache

Reply:
Will add a todo

Reply:
Sure 👍